cfme/web_ui/toolbar.py

```diff
@@ -146,9 +146,17 @@ def old_select(root, sub=None, invokes_alert=False):
     else:
         search = root_loc(root)
 
-    try:
-        idd = sel.get_attribute(search, 'idd')
-    except sel.NoSuchElementException:
+    eles = sel.elements(search)
+
+    for ele in eles:
+        try:
+            idd = sel.get_attribute(ele, 'idd')
+            if not idd:
+                continue
+            break
```

Comment:
This block could be just:
```
idd = sel.get_attribute(ele, 'idd')
if idd:
    break
```
without the `continue`.
Also - I dont believe this has to be wrapped in a `try-except` block because `.get_attribute` on an existing element (and we are cycling over existing elements - if not, we want to know about it) will return None for non-existing attributes, it won't be raising anything.